### ISSUE TITLE
User move is skipped

### ISSUE THREAD

**Issue by yanbronshtein** [hernanpesantez/355-project#2]
User move is skipped when they press on a cell taken by the AI

**Comment by hernanpesantez** [comment]
Yes, that's something we have to work on. that would have to be implemented in the frontend. with an if statement or try and catch.
